docs/machine-learning/how-to-guides/getting-started-dataframe.md

```diff
@@ -0,0 +1,128 @@
+4,850f,150000,205000
+5,900f,155000,210000
+6,550f,99000,180000
+```
+
+Start by loading the data into a DataFrame. 
+
+```csharp
+using System.IO;
+using System.Linq;
+using Microsoft.Data.Analysis;
+
+// Define data path
+var dataPath = Path.GetFullPath(@"housing-prices.csv");
+
+// Load the data into the data frame
+var dataFrame = DataFrame.LoadCsv(dataPath);
+```
+
+# Inspect Data (Info/Describe)
+
+DataFrames store data as a collection of columns. This makes it easy to interact with the data. 
+
+To get a preview of the column datatypes, run `Info()`.
+
+```csharp
+dataFrame.Info();
+```
+
+To get a preview of what the data looks like, run `Description()`.
+
+```csharp
+dataFrame.Description()
+```
+# Transform Data
+
+There are a variety of transformative options for data. The [DataFrame](https://learn.microsoft.com/dotnet/api/microsoft.data.analysis.dataframe?view=ml-dotnet-preview) and [DataFrameColumn](https://learn.microsoft.com/dotnet/api/microsoft.data.analysis.dataframecolumn?view=ml-dotnet-preview) classes expose a number of useful APIs: binary operations, computations, joins, merges, handling missing values and more.
+
+For example, this data can be edited to compare historical prices to current prices accounting for inflation. We can easily apply a computation to all of the values. 
+
+```csharp
+dataFrame["ComputedPrices"] = dataFrame["HistoricalPrice"].Multiply(2);
+```
+
+Data can be sorted into groups. 
+
+```csharp
+var sortedDataFrame = dataFrame.GroupBy("Size");
+```
+
+Data can be filtered based on different equality metrics. This example uses a ElementWise equality function, and then filters based on the boolean result column to get a new DataFrame with only the appropriate values. 
+
+```csharp
+PrimitiveDataFrameColumn<bool> boolFilter = dataFrame["CurrentPrice"].ElementwiseGreaterThan(200000);
+DataFrame filteredDataFrame = dataFrame.Filter(boolFilter);
+```
+
+# Combine Data Sources
+
+```text
+Id, Bedrooms	
+1, 1	
+2, 2	
+3, 3	
+4, 2	
+5, 3	
+6, 1
+```
+DataFrames can be constructed from individual data columns. Create a DataFrame from a list of the raw data above. 
+
+```csharp
+var ids = new List<Single>() {1,2,3,4,5,6};
+var bedrooms = new List<Single>() {1, 2, 3, 2, 3, 1};
+
+var idColumn = new SingleDataFrameColumn("Id", ids);
+var bedroomColumn = new SingleDataFrameColumn("BedroomNumber", bedrooms);
+var dataFrame2 = new DataFrame(idColumn, bedroomColumn);
+```
+
+The two DataFrames can be merged based on the Id value. The merge function will take both DataFrames, and combine rows based on their id. 
+
+```csharp
+dataFrame = dataFrame.Merge(dataFrame2, new string[] {"Id"}, new string[] {"Id"});
+```
+
+# Save DataFrames
+
+Results can be saved back into a .csv format. 
+
+```csharp
+DataFrame.WriteCsv(dataFrame, "result.csv", ',');
+```
+
+# Use DataFrame with ML.NET
+
+DataFrames can also be easily used by ML.NET. DataFrame implements the IDataView and can be used directly to train a model. 
```

Comment:
Thoughts on this? I wasn't sure if I should link to the next steps doc? 

You can just use the DataFrame directly as the IDataView (as long as no random bugs pop up!), but I'm not sure how to show that to customers.

Reply:
This would be a doc to point people to.

https://learn.microsoft.com/dotnet/machine-learning/how-to-guides/train-machine-learning-model-ml-net